cmd/ingest/main.go

```diff
@@ -1,18 +1,23 @@
 package main
 
 import (
+	"context"
 	"database/sql"
-	"encoding/binary"
 	"encoding/hex"
+	"encoding/json"
 	"flag"
 	"fmt"
 	"os"
+	"time"
```

Comment:
Many of the changes in this Pull Request seem to originate from commits on the `remove_0mq` branch of in @mdr0id 's fork. So we may want to restructure this PR so that it builds ontop of that work so that it's easier to track the true history in the repository.

This would mean, @mdr0id would submit a PR to this repo and then, once that's merged, this PR can rebase on top of those changes.